src/argilla/training/base.py

```diff
@@ -46,6 +46,7 @@ def __init__(
         train_size: Optional[float] = None,
         seed: Optional[int] = None,
         gpu_id: Optional[int] = -1,
+        framework_kwargs: Optional[dict] = {},
```

Comment:
@alvarobartt For me it makes more sense to add this to the `update_config()`.

Reply:
As we can also define the `optimize` value there I'd keep it as is, plus in an upcoming iteration moving the `spaCy`-specific stuff we have there to the `framework_kwargs` arg, to be provided in the `__init__`